### ISSUE TITLE
Seek to functionality?

### ISSUE THREAD

**Issue by eminos** [arielfaur/ionic-audio#1]
This module is awesome. Exactly what i was looking for. Thank you.
I see that you have the "seek to via slider" functionality on your todo list. Do you have any code for this already, or maybe an alpha version, or an ETA? :)

**Comment by arielfaur** [comment]
Thanks! I am glad you liked it. It was a bit of a challenge to put the pieces together to work. I am actually working on refactoring parts of the module. But in the meantime I have looked at the seek-to functionality which is also a bit of a challenge. I think I found an approach that could work. Will get back to you soon :)

**Comment by eminos** [comment]
I'm a real newbie at angular, otherwise I would help you. It's a nice little project. Maybe later on once I get more comfortable with angular.
Thanks for looking into seek-to!

**Comment by arielfaur** [comment]
@eminos i got the seek-to working today :)  

Latest version is 1.2.1:
`bower install ionic-audio#1.2.1`
There is a minor change to the directives. There is a single `ion-audio-progress-bar` that works both as embedded and global, so it is even simpler now. I have updated the documentation, but basically replace `ion-audio-progress-box` with `ion-audio-progress-bar` in case you were using it.
Please let me know if you find any issues. Are you on iOS by the way? I am curious whether it works at all.

**Comment by eminos** [comment]
Awesome! I will try it out this weekend.
I'm on Android, but the final app I'm building will be on iOS as well, so I will test that in about 2-3 weeks.

**Comment by ascendedmasta** [comment]
This is great! I was actually messing with something similar a few months back, but I got busy with some other projects. I managed to get a Progress bar/Seek To working too, but it looks like your is much cleaner. Actually the whole module is much cleaner. Good job!

**Comment by ascendedmasta** [comment]
Also, I have your example running on iOS on three devices: iPhone 6, iPhone 5, iPad Mini 2. :) iOS 8.3 and above. Hoping I can get it on a Note 4 soon, too.

**Comment by arielfaur** [comment]
@ascendedmasta thanks for your feedback! Good to know that it works on iOS too,
Regarding the progress bar, that is a bug. Thanks for reporting! Can you open a separate issue?
The intended use case was to disabled it by default and enable it only once the media has been loaded.

**Comment by ascendedmasta** [comment]
Sure thing, I'll open another issue.
